app/views/posts/show.html.erb

```diff
@@ -1,18 +1,29 @@
 <section class="post">
   <div class="post__detail">
     <div class="post__detail--body">
-      <h3>Post #<%= @post.id%> by <%= @post.user.name %></h3>
+      <h3>Post #<%= @post.id %> by <%= @user.name %></h3>
       <div class="post__detail--comment">
-        <span class="post__detail--com">Comments: <%= @post.comments_counter || 0%>, </span>
+        <span class="post__detail--com">Comments: <%= @post.comments_counter || 0 %>, </span>
         <span class="post__detail--likes">Likes: <%= @post.likes_counter || 0 %> </span>
       </div>
     </div>
     <p class="post__detail--text"> <%= @post.text %> </p>
+
+    <div>
+      <%= form_with model: @like, url:  user_post_likes_path(@user, @post) do |form|%>
+          <%= form.submit 'Like post', class: 'submit' %>
+        <%end%>
+    </div>
   </div>
+
   <ul class="post__detail--list">
     Comments:
     <% @comments.each do |comment| %>
-      <li class="post__detail--li"><%= comment.user.name%>: <%= comment.text %></li>
+      <li class="post__detail--li"><%= @user.name%>: <%= comment.text %></li>
```

Comment:
- [ ] As mentioned in the previous review, kindly change the `user` to `author`

Reply:
Hello @addod19 
I got this error when I changed the code to `author`. How do I fix it?
![Screenshot (6)](https://github.com/Ogaga01/Blog-App/assets/98261268/1690044c-43ad-4687-8f78-0fcacdaf515d)

Reply:
Hello @Ogaga01, if you want to rename the `user` to `author` you need to run some migrations, renames the associations and reset your database. hint 
```rb 
# change user_id for posts or comments
class ChangeUserToAuthor < ActiveRecord::Migration
   def change
      rename_column :posts, :user_id, :author
   end
end

Reply:
Hello @andersonlebon , I have renamed the user_id columns in the posts and comments table to author_id. If you check the post/index.html.erb file, you'll see that the author's name is being rendered correctly. But it isn't for the show single post view.

Reply:
@Ogaga01  let me have a look

Reply:
@andersonlebon please let me know when you do

Reply:
@Ogaga01  
<img width="950" alt="image" src="https://github.com/Ogaga01/Blog-App/assets/65068771/dcca6f76-1828-4473-970d-5ad29f57efe4">


Solve it by using just the `comment.author` not `comment.author.name`. hint 👇 '

```html.erb
<ul class="post__detail--list">
    Comments:
    <% @comments.each do |comment| %>
      <li class="post__detail--li"><%= comment.author%>: <%= comment.text %></li>
    <% end %>
  </ul>
```

Reply:
@andersonlebon It's done. Thank you.